### DISCUSSION TITLE
openDTU Anzeigen Gesamtertrag "heute" und "insgesamt" sind vom Wert gleich, was falsch ist

### DISCUSSION THREAD

**Discussion by sschoeppi** [tbnobody/OpenDTU#1149] [Q&A]
Hallo liebe Community,
erst mal, ich bin neu hier und hoffe alles korrekt zu posten. Falls nicht, bitte ich um Nachsicht.  Kurz zu mir, damit mein Background etwas klarer wird.
Ich beschäftige mich seit knapp einem Jahr mit PV und bin so in Richtung Smarthome gerutscht. Mittlerweile habe ich ioBroker, einige Shellies und Tasmota Steckdosen zum Laufen gebracht und habe richtig Spaß an der Automatisierung gefunden.
Hier eine Frage in die Runde, da ich nicht weiter komme.
Ich habe die OpenDTU (FW v23.6.1) zum Laufen gebracht und in IoBroker integriert. Soweit alles O.K.
Allerdings sind folgende Energieanzeigen falsch. Das sieht in einer Momentaufnahme so aus mit Faktor 1000 dazwischen.
Wert in der OpenDTU:
   - Gesamtertrag insgesamt 44,695 kWh
   - Gesamtertrag heute: 44.695 Wh
Wert im IoBroker:
   - yieldtotal : 44695
   - yieldday : 44,695
Der Tagesertrag sollte irgendwas zwischen 1000 bis 2000Wh anzeigen, dass ist das, was ich nachts einspeise aus einer Batterie. 
Ob der Gesamtertrag stimmt, habe ich noch nicht versucht nachzuvollziehen. 
Habt Ihr eine Idee was ich falsch mache? 
Es scheint kein generelles Problem zu sein, sonst hätte ich vermutlich schon einen Beitrag dazu gefunden.

vg sschoeppi

**Comment by Bernhard59** [comment]
Hallo zusammen... ja, so etwas ähnliches habe ich auch gehabt. Dann habe ich aber festgestellt, dass der "Gesamtertrag Heute" bei einem Neustart des Wechselrichters zurückgesetzt wird. Der Gesamtertrag ist so eine Art "Kilometerzähler" des Gerätes, der nicht zurückgesetzt werden kann. Im Moment lasse ich meinen Wechselrichter jede Nacht 00:00 Uhr neu starten. Damit ist zwar der "Gesamtertrag Heute" täglich aktuell, aber ich weiß nicht ob das gut für den Wechselrichter ist und ob es vielleicht auch noch eine andere Möglichkeit gibt, den Wert täglich zurück zu setzten... Gruß Bernhard

**Reply by kailausberg** [reply]
Ich habe eine ähnliche Konfig, iobroker, openDTU Adapter usw...  Ich mache keinen Neustart des Wechselrichters, bei mir setzt sich der Tageszähler von alleine morgens zurück, nachdem sich der Wechselrichter aufgeweckt hat bei Sonnenaufgang. Bis dahin bleibt der alte Tageszähler vom Vortag erhalten in der Anzeige.

**Reply by Bernhard59** [reply]
Hallo, ja... aber bei mir hängt der Wechselrichter ja an einer 24V 100Ah LiFePO4 Batterie und wird somit rund um die Uhr eingespeist. Darum ergibt sich ein unendlich langer Tag...

**Reply by kailausberg** [reply]
ah ok, verstehe nun ;)  Offenbar bleibt dann nur der Neustart des Inverters zum Zurücksetzen des Tagesertrages.

**Reply by DPO99** [reply]
Es gibt stand jetzt nur den Neustart. Generell startet der WR ja eh jeden Tag neu, wenn die Sonne weg ist. Ob der "Software" Neustart schädlicher ist kann ich dir allerdings nicht sagen. Zur Not kann man ja aber im IoBroker ja auch jeden Tag berechnen, in dem man den Gesamtertrag täglich speichert und gegeneinander verrechnet (Aufwand ist ja super gering). Ich lasse meine Hoymiles z.B. auch durchlaufen und zähle den Tagesertrag eben anders.

**Comment by sschoeppi** [comment]
Hallo , sorry für das späte Feedback Ich war in Urlaub :-)
Erst mal danke für das Feedback. Bei mir ist es auch so, dass der WR immer "am Netz" ist (Batterie) und nicht neu gestartet wird. Ich schalte die Sekundärseite tagsüber ab, bzw. abends über Steckdosensteuerung zu. So gesehen passt das bei mir vermutlich auch mit den Aussagen (@Bernhard59) überein, dass es einen Neustart geben muss damit der Tageswert korrekt angezeigt wird. Im IoBroker den Tageswert zu bilden geht natürlich auch. Ich wollte es aber auch gerne mal spontan auf der OpenDTU Seite sehen. Nebenbei, mein Gesamtertrag hat sich in den letzten 2 Wochen währenn des Urlaubs irgendwann genullt, da die kWh Anzeige nun bei ca.5,5 liegt... vg
